pkg/minikube/reason/reason.go

```diff
@@ -460,4 +460,25 @@ var (
 		`),
 		Style: style.SeeNoEvil,
 	}
+
+	NotFoundCriDockerD = Kind{
```

Comment:
nit: Based on the official capitalization of `Systemd` I think it makes sense for the trailing `D` to be lowercase.

Reply:
Sounds good, applied suggestion.